pivoting/script.sh

```diff
@@ -1,30 +1,31 @@
 #!/bin/bash
 TARGET_CLUSTER=192.168.111.21
 # Source cluster: collect kubeconfig for source and target clusters.
-cd ~/metal3-dev-env/
+cd ~/metal3-dev-env/ || exit
 cp ~/.kube/config source.yaml
 scp ubuntu@"${TARGET_CLUSTER}":/home/ubuntu/.kube/config target.yaml
 # Source cluster: create a text file from lease files
-export bmo_pod=$(kubectl  get pods -n metal3 -o name | grep 'metal3-baremetal-operator' | cut -f2 -d'/')
-kubectl exec -n metal3 $bmo_pod -c ironic-dnsmasq cat /var/lib/dnsmasq/dnsmasq.leases > /tmp/dnsmasq.leases
+bmo_pod="$(kubectl  get pods -n metal3 -o name | grep 'metal3-baremetal-operator' | cut -f2 -d'/')"
+export bmo_pod
+kubectl exec -n metal3 "$bmo_pod" -c ironic-dnsmasq cat /var/lib/dnsmasq/dnsmasq.leases > /tmp/dnsmasq.leases
 # Target cluster: Create config from a lease file content | commands run on the source cluster remotely.
 scp /tmp/dnsmasq.leases ubuntu@"${TARGET_CLUSTER}":/tmp/dnsmasq.leases
 ssh ubuntu@"${TARGET_CLUSTER}" -- kubectl create ns metal3
 ssh ubuntu@"${TARGET_CLUSTER}" -- kubectl create configmap dnsmasq-leases-configmap --from-file=dnsmasq.leases=/tmp/dnsmasq.leases -n metal3
 ssh ubuntu@"${TARGET_CLUSTER}" -- kubectl get configmap dnsmasq-leases-configmap -o yaml -n metal3
 # Source cluster: Start pivoting
-cd ~/go/src/sigs.k8s.io/cluster-api/cmd/clusterctl/
+cd ~/go/src/sigs.k8s.io/cluster-api/cmd/clusterctl/ || exit
 alias pivot='./clusterctl alpha phases pivot -p ~/go/src/sigs.k8s.io/cluster-api/provider-components-target.yaml  -s ~/metal3-dev-env/source.yaml -t ~/metal3-dev-env/target.yaml -v 5'
```

Comment:
Not part of the change, but defining an alias in the middle of the script, and then using it exactly once? Ouch.